public/static/partials/project_trigger_modal.html

```diff
@@ -0,0 +1,65 @@
+<md-dialog style="width:650px">
+    <md-dialog-content>
+        <form ngSubmit="addTrigger()">
+            <md-card style="width:97.5%">
+                <md-card-title>
+                    <div ng-bind="modalTitle()"></div>
+                </md-card-title>
+                <md-card-content>
+                    <section layout="row" flex>
+                        <md-input-container flex=50>
+                            <label>Project:</label>
+                            <input type="text" ng-model="trigger.project" ng-required="true">
+                        </md-input-container>
+                        <md-input-container flex=50>
+                            <label>Date cutoff:</label>
+                            <input type="number" step="1" min="0" ng-model="trigger.date_cutoff" placeholder="Commits older than # days will not trigger">
+                        </md-input-container>
+                    </section>
+                    <section layout="row" flex>
+                        <md-input-container flex=50>
+                            <label>Level:</label>
+                            <md-select ng-model="trigger.level">
+                                <md-option value="task">Task</md-option>
+                                <md-option value="build">Build</md-option>
+                            </md-select>
+                        </md-input-container>
+                        <md-input-container flex=50>
+                            <label>Status:</label>
+                            <md-select ng-model="trigger.status">
+                                <md-option value="all">All</md-option>
+                                <md-option value="success">Success</md-option>
+                                <md-option value="failed">Failed</md-option>
+                            </md-select>
+                        </md-input-container>
+                    </section>
+                    <section layout="row" flex>
+                        <md-input-container flex=50>
+                            <label>Variant regex:</label>
+                            <input type="text" ng-model="trigger.variant_regex" placeholder="Variants to run">
+                        </md-input-container>
+                        <md-input-container flex=50>
+                            <label>Task regex:</label>
+                            <input type="text" ng-model="trigger.task_regex" placeholder="Tasks to run">
+                        </md-input-container>
+                    </section>
+                    <section layout="row" flex>
+                        <md-input-container flex=50>
+                            <label>Command:</label>
+                            <input type="text" ng-model="trigger.command" placeholder="Command that creates task json file">
```

Comment:
I feel like it's not clear that you should put json/yaml here? or is this supposed to be a shell command?

Reply:
nit: put the word shell in the ghost text.